config/sys.config

```diff
@@ -1,6 +1,15 @@
 [
     {hpr, [
-        {base_dir, "/var/data"}
+        {base_dir, "/var/data"},
```

Comment:
Should we remove this file?
I don't think there's ever a time rebar3 will use it when there's a `.src` file available.